backend/src/main/java/com/cruru/email/util/FileUtil.java

```diff
@@ -4,17 +4,26 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.web.multipart.MultipartFile;
 
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
```
private 생성자를 lombok을 사용해서 통일하는 것은 어떨까요~?